mint/install-python/mint-leaf.yml

```diff
@@ -39,36 +39,21 @@ tasks:
 
       # Download and extract the prebuilt Python
       echo "Downloading Python $PYTHON_VERSION..."
-      curl -o layer.tar.gz -fsSL "https://mint-prebuilt-binaries20240129200014482300000001.s3.us-east-2.amazonaws.com/mint/install-python/$path"
-      echo "$sha256sum layer.tar.gz" | sha256sum --check --status
+      curl -o layer.tar -fsSL "https://mint-prebuilt-binaries20240129200014482300000001.s3.us-east-2.amazonaws.com/mint/install-python/$path"
+      echo "$sha256sum layer.tar" | sha256sum --check --status
       echo "Downloaded Python $PYTHON_VERSION!"
 
-      echo "Applying whiteouts..."
-      whiteouts=$(tar -tf layer.tar.gz | { grep -E "(^\.wh\.[^/]+)|(/\.wh\.[^/]+)" || test $? = 1; })
-
-      # whiteout specific files
-      non_opaque_whiteouts=$(echo "$whiteouts" | { grep -v ".wh..wh..opq" || test $? = 1; })
-      files_to_whiteout=$(echo "$non_opaque_whiteouts" | sed -E "s/^\.wh\.([^/]+)/\1/g" | sed -E "s/\.wh\.([^/]+)/\1/g")
-      echo "$files_to_whiteout" | xargs -I{} sudo rm -rf "/{}"
-
-      # whiteout opaque siblings
-      opaque_whiteouts=$(echo "$whiteouts" | { grep ".wh..wh..opq" || test $? = 1; } | xargs -I{} dirname {})
-      echo "$opaque_whiteouts" | xargs -I{} sudo rm -rf "/{}/..?*" "/{}/.[!.]*" "/{}/*"
-      echo "Applied whiteouts!"
```

Comment:
I'm _pretty_ sure it's safe to omit all the whiteout logic and simply write the 0,0 character devices to the layer as part of the task, but I'm going to verify separately prior to merging

Reply:
Narrator: he was incorrect